.travis.yml

```diff
@@ -26,3 +32,12 @@ notifications:
       - cc-devel@lists.01.org
     on_success: change
     on_failure: always
+
+addons:
+  coverity_scan:
+    project:
+      name: "01org/cc-oci-runtime"
+      description: "Build submitted via Travis CI"
+    build_command_prepend: "./autogen.sh --enable-cppcheck --enable-valgrind; make clean"
+    build_command: "make -j 4"
```

Comment:
That made me look at what travis provides:

https://docs.travis-ci.com/user/ci-environment/#Virtualization-environments

2 cores, so maybe we can even push that to 5? Doing it for the main build looks useful as well.